src/connector/src/parser/avro/schema_resolver.rs

```diff
@@ -50,12 +56,14 @@ impl ConfluentSchemaCache {
         }
     }
 
-    pub async fn get_by_subject_name(&self, subject_name: &str) -> ConnectorResult<Arc<Schema>> {
-        let raw_schema = self.get_raw_schema_by_subject_name(subject_name).await?;
+    /// Non-cached
```

Comment:
The `parse_and_cache_schema` call below caches it.

Reply:
I had some misunderstanding of the API. Just further improved them.